src/SubCommands/SubCommand.php

```diff
@@ -98,11 +98,11 @@ private function setFilePath() : string {
     }
 
     /**
-     * Reads rule template file. Depending on output type, returns string or an array
+     * Reads rule template file. Depending on output type, returns an array
      *
-     * @return string|array
+     * @return array
      */
-    private function setRuleContent() : string|array {
+    private function setRuleContent() : array {
```

Comment:
This method can return both string and an array, just remove multiple types as output type.